src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ToGerritRunListener.java

```diff
@@ -428,15 +429,18 @@ private String obtainFailureMessage(@Nullable GerritTriggeredEvent event,
                 // The filename may contain environment variables
                 filepath = envVars.expand(filepath);
 
-                // Check for ANT-style file path
-                FilePath[] matches = this.getMatchingWorkspaceFiles(build.getWorkspace(), filepath);
-                logger.debug("Found matching workspace files: {}", matches);
-
-                if (matches.length > 0) {
-                    // Use the first match
-                    FilePath path = matches[0];
-                    content = this.getExpandedContent(path, envVars);
-                    logger.info("Obtained failure message from file: {}", content);
+                // TODO: what will we do here?
```

Comment:
There is no `getCurrentExecutor` anywhere in Jenkins core, so I am not sure what you could mean by that.

Currently there is nothing else to be done here except check for an instance of `AbstractBuild`. [JENKINS-26138](https://issues.jenkins-ci.org/browse/JENKINS-26138) might involve introducing a core API whereby you could enumerate workspaces associated with a `Run`.

In the meantime, it seems reasonable to just fall back to a message that does not rely on knowledge of workspace files.